CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
+  Implements [Specify resource attributes using Kubernetes annotations](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md#specify-resource-attributes-using-kubernetes-annotations).
+
+  If you are using the file log receiver, you can now create the same resource attributes as traces (via OTLP) received
+  from an application instrumented with the OpenTelemetry Operator -
+  simply by adding the `extract: { otel_annotations: true }` configuration to the `k8sattributesprocessor` processor.
+  See the [documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/README.md#config-example) for more details.
+
+- (Contrib) `oracledbreceiver`: Add support for parallel operations metrics ([#39215](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39215))
+  The following metrics are now available, all disabled by default:
+  - 'DDL statements parallelized'
+  - 'DML statements parallelized'
+  - 'Parallel operations not downgraded'
+  - 'Parallel operations downgraded to serial'
+  - 'Parallel operations downgraded (1-25%)'
+  - 'Parallel operations downgraded (25-50%)'
+  - 'Parallel operations downgraded (50-75%)'
+  - 'Parallel operations downgraded (75-99%)'
+
+- (Contrib) `k8sclusterreceiver`: Add missing attributes to entities in experimental entity feature ([#39038](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39038))
+- (Contrib) `pkg/stanza`: Use buffer pool for the read buffers to limit allocations ([#39373](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39373))
+- (Contrib) `postgresqlreceiver`: add top query collection to help end user identify which query were executed in the postgresql database. ([#39311](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39311))
+- (Contrib) `sqlserverreceiver`: Allow full control of the "connection string" via the `datasource` configuration option ([#39235](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39235))
+- (Contrib) `spanmetricsconnector`: Initialise new calls_total metrics at 0 ([#38537](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/38537))
+- (Contrib) `pkg/stanza`: Remove unnecessary slice allocation to track errors (even nil) ([#39367](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39367))
+
+### 🧰 Bug fixes 🧰
+
+- (Core) `confighttp`: Ensure http authentication server failures are handled by the provided error handler ([#12666](https://github.com/open-telemetry/opentelemetry-collector/issues/12666))
+- (Contrib) `receiver/filelog`: Remove printing to console the names of files which are read by File Log receiver ([#39414](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39414))
```

Comment:
```suggestion
```
Nit: Our repo wasn't impacted by this bug. It was first introduced in `v0.124.0` in `contrib`, and fixed in contrib's `v0.124.1`. Since we didn't have any release depending on `v0.124.0` we're never hit.

This isn't a big deal, but this bug specifically was what required `v0.124.1` of contrib to be released.

Reply:
Good point. I'll remove this